src/tests/execute/create_ask_tests.rs

```diff
@@ -64,14 +69,27 @@ mod create_ask_tests {
             },
         );
 
-        // TODO: find alternative function
-        // deps.querier.with_attributes(
-        //     "asker",
-        //     &[
-        //         ("ask_tag_1", "ask_tag_1_value", "String"),
-        //         ("ask_tag_2", "ask_tag_2_value", "String"),
-        //     ],
-        // );
+        QueryAttributeRequest::mock_response(
```

Comment:
Tests with these are failing with the following error. I'm still trying to fix this. Any ideas?
```
unexpected error: Std(GenericErr { msg: "Querier system error: Unsupported query type: /provenance.attribute.v1.Query/Attributes" })
```